index.bs

```diff
@@ -5,8 +5,8 @@ URL: https://AOMediaCodec.github.io/av1-avif
 Shortname: av1-avif
 Editor: Cyril Concolato, Netflix, cconcolato@netflix.com
 Former Editor: Paul Kerr, Netflix, pkerr@netflix.com
-Abstract: This document specifies syntax and semantics for the storage of [[!AV1]] images in the generic image format [[!HEIF]], which is based on [[!ISOBMFF]]. While [[!HEIF]] defines general requirements, this document also specifies additional constraints to ensure higher interoperability between writers and readers when [[!HEIF]] is used with [[!AV1]] images. These constraints are defined in the form of a [[!MIAF]] profile.
-Date: 2018-08-29
+Abstract: This document specifies syntax and semantics for the storage of [[!AV1]] images in the generic image file format [[!HEIF]], which is based on [[!ISOBMFF]]. While [[!HEIF]] defines general requirements, this document also specifies additional constraints to ensure higher interoperability between writers and readers when [[!HEIF]] is used with [[!AV1]] images. These constraints are defined in the form of profiles for the Multi-Image Application Format [[!MIAF]].
```

Comment:
I don't understand what you mean by "for" in "they are actual profiles for the MIAF format".  The MIAF spec says "External organizations may define additional MIAF based profiles". That is what our spec is doing. "extend" can be confusing as it could mean that the profiles defined in AVIF extend the profiles defined in MIAF. What about:
> profiles based on constraints defined in the Multi-Image Application Format.